packages/grafana-ui/src/components/DataSourceSettings/DataSourceHttpSettings.tsx

```diff
@@ -159,223 +159,211 @@ export const DataSourceHttpSettings = (props: HttpSettingsProps) => {
       );
   }
 
-  const accessSelect = (
-    <Select
-      aria-label="Access"
-      className="width-20 gf-form-input"
-      options={ACCESS_OPTIONS}
-      value={ACCESS_OPTIONS.filter((o) => o.value === dataSourceConfig.access)[0] || DEFAULT_ACCESS_OPTION}
-      onChange={(selectedValue) => onSettingsChange({ access: selectedValue.value })}
-      disabled={dataSourceConfig.readOnly}
-    />
-  );
-
   const isValidUrl = /^(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/.test(
     dataSourceConfig.url
   );
 
-  const notValidStyle = css({
-    boxShadow: `inset 0 0px 5px ${theme.v1.palette.red}`,
+  const gridLayout = css({
```

Comment:
Can we use the Grid component instead?

Reply:
Nope, it seems to lack auto width for the column. Maybe this is a feature we need?